examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't thnik we really need this example given that we have the "hole" one above